### ISSUE TITLE
Update Guide/Installation Guide

### ISSUE THREAD

**Issue by Kaldarash** [googleads/googleads-mobile-unity#288]
I haven't "installed" the latest plugin for a long time, but I can't seem to find the installation guide and I seem to recall having to drop extra files or something into some folder. Is such a thing still required, or is it fine to delete the old folder and import the unity package and call it a day? I imported over the old one since I remembered having to place files in places and it's working, but I've gotten the same ad for about 30 minutes now - it keeps serving it to me, I see it disappear and reappear, sometimes as the alt color. I know it's typically all about who is paying and it can serve you the same ad, but the sheer amount and timing of the odd occurrence lead me to believe that maybe something is up. 

So, tl;dr, I'm wondering if there's a link to a guide on installing or updating the plugin, and make a request to include the information in the readme.

**Comment by rampara** [comment]
The latest integration instructions can be found in our [developer docs](https://firebase.google.com/docs/admob/unity/start). You no longer have to include the Google Play services library project within your Unity project. Removing it and using the latest plugin release should work fine.

**Comment by ForestJungle** [comment]
Hello Rampara .Before I asked you about rewared video admob .I wana say with you that Reward video Documents is incomplete.
This link which document write: https://firebase.google.com/docs/admob/unity/rewarded-video
Document don't indicate anyway I should add more Apdater and SDK Adwork and I should config it how.
with document about video I can't really do it.

**Comment by rampara** [comment]
The developer docs have been updated with a new page on how to set up [mediation](https://firebase.google.com/docs/admob/unity/mediation). Also, AdMob demand for rewarded video is now publicly available so you can request and display rewarded video ads without setting up mediation.
